src/main/resources/api.yaml

```diff
@@ -207,3 +270,15 @@ definitions:
     required:
       - enrollmentPeriod
       - durationPeriod
+
+  UserAvailabilityDto:
+    type: object
+    properties:
+      sprintDurationPeriod:
```

Comment:
Po co nam ta informacja tutaj?

Reply:
na widoku miało się coś takiego wyświetlać

Reply:
to wtedy powinieneś zrobić request na sprint chyba?

Reply:
no można i tak, mi to w sumie bez różnicy

Reply:
Mi się też wydaje, że lepiej osobno